templates/types/streaming/fastapi/app/api/routers/chat.py

```
@@ -40,12 +45,35 @@ async def chat(
+            filters=filters,
+            params=params,
+            event_handlers=[event_handler],
+            chat_history=messages,
         )
-        response = await chat_engine.astream_chat(last_message_content, messages)
-        process_response_nodes(response.source_nodes, background_tasks)
 
-        return VercelStreamResponse(request, event_handler, response, data)
+        if isinstance(chat_engine, CondensePlusContextChatEngine) or isinstance(
+            chat_engine, AgentRunner
+        ):
```

Ruff

Multiple `isinstance` calls for `chat_engine`, merge into a single call

Merge `isinstance` calls for `chat_engine`

(SIM101)